woocommerce/woocommerce-android
# Send code coverage reports to CodeCov

Closes: #6101 


### Description

This PR adds integration of CodeCov tool.

### Testing instructions

You can navigate to https://app.codecov.io/gh/woocommerce/woocommerce-android and browse statistics but keep in mind that there's no yet report for `trunk` so you might experience some 404s or other strange behaviour (CodeCov has nothing to compare with).

Also this PR didn't change any code that is meant to be covered. For testing purposes I've added (and later removed) some test commit. Please see it here: https://app.codecov.io/gh/woocommerce/woocommerce-android/commit/96f3e62278c56968c34ffe008ac8a39bdc67b986

### Images/gif


<img width="1158" alt="image" src="https://user-images.githubusercontent.com/5845095/160441012-9a704298-b443-4300-9a52-083509844d81.png">
